lms/services/user.py

```diff
@@ -84,7 +83,6 @@ def factory(_context, request):
     """Service factory for the UserService."""
 
     return UserService(
-        request.find_service(name="application_instance"),
         request.db,
         request.registry.settings["h_authority"],
     )
```

Comment:
```suggestion
    return UserService(request.db, request.registry.settings["h_authority"])
```

I find it's always worth knocking off the trailing commas to see if `black` will back things up onto a single line.

Reply:
Github didn't allow me to commit these from here but just pushed them :+1: